### PR TITLE
jupyter-no-output.rs

### PR DESCRIPTION
**Pull request by AaronM04**
This is the pre-commit hook for the `conwayste` repo.

### CODE REVIEW

File: jupyter-no-output/jupyter-no-output.rs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+//!
+//! ```cargo
+//! [dependencies]
+//! anyhow = "1.0.*"
+//! serde_json = "1.0.*"
+//! ```
+
+use std::process::{Command, ExitCode};
+
+use anyhow::{self as A, bail};
```
Comment (Manghi) [inline jupyter-no-output/jupyter-no-output.rs]: Why use the abbreviation `A`? Seems to read clearer if the `anyhow` is kept as is.

**Review by Manghi — CHANGES_REQUESTED** [review]
The big suggestion would be to explicitly handle and return an exit code where `expect()` is used. Seems more reasonable than just panicking.

**Comment by AaronM04** [comment]
> The big suggestion would be to explicitly handle and return an exit code where expect() is used. Seems more reasonable than just panicking.

Sure, but that would take some time, and I just wanted to quickly get this in a repo. Given the context of the code (a commit hook), it seems fine to me to panic.

File: jupyter-no-output/jupyter-no-output.rs (hunk 1 above)
Comment (AaronM04) [reply jupyter-no-output/jupyter-no-output.rs]: Thanks, I'll change it.

**Review by Manghi — APPROVED** [review]
LGTM :D

**Pull request merged**